05-Organizing-and-Executing-Simulations/02-Executing-Simulations.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>The operation doesn't necessarily take input files (even if the statepoint and document are files users do not interact with them as such). An operation is an operation that updates the passed job's workspace.</p><p> </p><p>Also, are you comparing the purpose of both packages' concept of operation? If not, I would change the second sentence to read something like "These steps have the same name as a ...".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I am attempting to point out to users that the word "operation" here may have one of two meanings depending on context. To avoid this, I use the language "workflow step" throughout the tutorial but "operation" must appear in the code implementation using flow.</p>